packages/flagship/src/commands/init.ts

```diff
@@ -254,7 +254,10 @@ function initWeb(
 
   fs.copySync(
     path.flagship.resolve('../fsweb'), // only works in the monorepo
-    path.project.resolve('web')
+    path.project.resolve('web'),
+    {
+      dereference: true
```

Comment:
When `fsweb` package is linked to a local version the folder `fsweb` is a symlink.
When it was copied the old way the subsequent steps would fail because it's not possible to copy other files and folders into symlink.
This parameter force `copySync` to follow the symlink and copy the actual folder.